nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateFlowFile.java

```diff
@@ -76,7 +76,8 @@ public class GenerateFlowFile extends AbstractProcessor {
             .description("The size of the file that will be used")
             .required(true)
             .defaultValue("0B")
-            .addValidator(StandardValidators.DATA_SIZE_VALIDATOR)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
```

Comment:
The `DATA_SIZE_VALIDATOR` includes support for expression language, so is this change necessary?

Reply:
Without that validator within the property, the processor was invalid indicating:
"'File Size' validated against '${random():mod(101)}B' is invalid because 'File Size' is not a supported property or has no Validator associated with it"
As the property is in fact required, this is a reasonable validator to add.

Reply:
@markobean Reverting this change and running the unit test worked as expected. See `StandardValidators` line 370 where the validator handles the presence of expression language. Do you still see the validation error?

Reply:
Thanks for pointing this out. I missed the fact that `DATA_SIZE_VALIDATOR` supports EL. The error I saw was only when no validator was provided at all.
I reverted back to `DATA_SIZE_VALIDATOR` and tested in a running NiFi. All good.